### PR TITLE
[CON-466] Migrate `ContentHubArticleList` to TypeScript

### PR DESCRIPTION
**Pull request by beau-charman**
Addresses https://aesoponline.atlassian.net/browse/CON-466

### CODE REVIEW

File: src/components/ContentHubArticle/ContentHubArticle.types.ts [hunk 1]
```diff
@@ -4,24 +4,27 @@ import type { ImageProps } from '~/components/Image/Image.types';
...
-  isMenuItem?: boolean;
-  isReadMore?: boolean;
   longTitle?: string;
   onClick?: () => void;
   readingTime?: string;
   uri?: string;
   verticalThumbnail?: Image;
 };
 
+type ContentHubArticleProps = Article & {
```
Comment (MkMan) [inline src/components/ContentHubArticle/ContentHubArticle.types.ts:19]: Nice 👌

File: src/components/ContentHubArticleList/ContentHubArticleList.tsx [hunk 2]
```diff
@@ -1,75 +1,88 @@
...
+  const getArticle = (isOnTop: boolean, isOnLeft: boolean) => {
     const articleIndex = getArticleIndex(isOnTop, isOnLeft);
+
     return articles[count - articleIndex];
   };
 
   return (
     <section>
-      <Fragment>
+      <>
```
Comment (MkMan) [inline src/components/ContentHubArticleList/ContentHubArticleList.tsx:80]: Nice 👌

File: src/components/ContentHubArticleList/ContentHubArticleList.tsx [hunk 3]
```diff
@@ -1,75 +1,88 @@
...
 import styles from './ContentHubArticleList.module.css';
 
-const ContentHubArticleList = ({
+const ContentHubArticleList: ContentHubArticleListType = ({
   articles,
   className,
-  isFirstGroup,
-  pattern, // 0 means first image is horizontal
+  isFirstGroup = false,
+  pattern = 0,
```
Comment (MkMan) [inline src/components/ContentHubArticleList/ContentHubArticleList.tsx:11]: > // 0 means first image is horizontal
Is it possible to retain this comment somewhere? Maybe in the type declaration?

**Review by MkMan — APPROVED** [review]

**Review by zack76 — APPROVED** [review]

File: src/components/ContentHubArticleList/ContentHubArticleList.tsx (hunk 3 above)
Comment (beau-charman) [reply src/components/ContentHubArticleList/ContentHubArticleList.tsx:11]: whoops. nice pick up!

**Pull request merged**